### PR TITLE
[ADD] event_registration_mass_mailing

### PR DESCRIPTION
**Pull request by JavierIniesta**
This module add actions to send mass mailings and add to mailing lists from
event registrations tree view.

### CODE REVIEW

**Comment by JavierIniesta** [comment]
cc @rafaelbn

**Comment by rafaelbn** [comment]
Rebuilding for testing! cc @yajo @antespi @sergio-incaser @carlos-incaser @janetrp

**Comment by yajo** [comment]
1. Add tests please.
2. When I create a mass mailing from registrations view, no mass mailing appears in menu _Marketing > Mass mailing_.
3. When I create a mass mailing from previous menu, I have no option to choose event registration recipients.
4. Readme lacks usage instructions.

File: event_registration_mass_mailing/README.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+Mass mailing from events
+========================
+
+This module add actions to send mass mailings and add to mailing lists from
```
Comment (pedrobaeza) [inline event_registration_mass_mailing/README.rst:9]: s/add/adds

Comment (pedrobaeza) [inline event_registration_mass_mailing/README.rst:9]: s/and add to/and allows to add partners to

**Comment by pedrobaeza** [comment]
This should be in OCA/event, not here.

**Comment by JavierIniesta** [comment]
OK. I close this PR and open a new one in OCA/event applying your suggestions.

**Comment by rafaelbn** [comment]
But @pedrobaeza this add mass mailing functionality, that's why is here. All related with mass mailing is in social

**Comment by pedrobaeza** [comment]
Well, this links event with mass_mailing. I see better to put it on event specialized repo.

**Comment by janetrp** [comment]
I have tested this improvement
The path for testing is:
From Marketing/Events/Registrations select the registered partners to an event, press button More and select Mass Mailing to Partners. This is useful if you want to filter the registered contacts in an event with Unconfirmed Status to send them a reminder.
I miss thes option for adding to a mailing list from Customers View.

**Comment by rafaelbn** [comment]
@pedrobaeza in fact the name is incorrect, it must be mass_mailing_event_registration, we touch model=mail.mass_mailing also.

**Comment by pedrobaeza** [comment]
OK. Please rename then.

**Comment by dreispt** [comment]
From what I read above I concur with @pedrobaeza in that this would be better as an OCA/event module. I see this as adding the mas mailing feature to the Event app.

**Pull request closed without merging**